app/assets/stylesheets/sign_in.scss

```diff
@@ -0,0 +1,9 @@
+body#sign-in {
+  background-color: #4b5c5c;
+  background: image_url('sign_in_bg.png');
```

Comment:
Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores<br>Prefer double-quoted strings